Profiles/mp-DispenseRequest.xml

```diff
@@ -188,6 +188,9 @@
         <targetProfile value="http://hl7.org/fhir/StructureDefinition/Medication" />
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/mp-PharmaceuticalProduct" />
       </type>
+      <binding>
```

Comment:
These bindings will lead to a validation error: _This element has a Binding but the types [Reference] do not make this kind of constraint relevant_. I've added a ticket for this on the Forge backlog a few weeks ago, and it will probably be fixed in the next version.

Reply:
Removed